core/metacling/src/TCling.cxx

```diff
@@ -4964,6 +4964,26 @@ const char* TCling::TypeName(const char* typeDesc)
    return t;
 }
 
+static bool rMapHasPCM(const char* rootmapfile, cling::Interpreter* interp)
```

Comment:
Please rename to `requiresRootMap` and negate the return result. Also, we should have a fast path for the non-modules case. Eg:
```cpp
  if (!fCxxModules)
    return true;
```